### ISSUE TITLE
Regarding adding an example in tfa.optimizers.CyclicalLearningRate

### ISSUE THREAD

**Issue by sayakpaul** [tensorflow/addons#2461]
Hi folks. 

I wanted to add a usage example in [CyclicalLearningRate](https://www.tensorflow.org/addons/api_docs/python/tfa/optimizers/CyclicalLearningRate) to make it easier for people to use this. 

I searched through [addons/docs](https://github.com/tensorflow/addons/tree/master/docs) but could not find the right place to determine where I could add it and submit a PR.

Help?

**Comment by seanpmorgan** [comment]
Hi @sayakpaul tutorials should be PR'd to this directory: 
https://github.com/tensorflow/addons/tree/master/docs/tutorials

Let us know if you encounter any issues with the Colab PR process (See README). Thanks!

**Comment by sayakpaul** [comment]
I actually want to submit a PR modifying the docs of [CyclicalLearningRate](https://www.tensorflow.org/addons/api_docs/python/tfa/optimizers/CyclicalLearningRate). The modification would include an example usage. That's it.

**Comment by seanpmorgan** [comment]
They're auto generated from the module:
https://github.com/tensorflow/addons/blob/master/tensorflow_addons/optimizers/cyclical_learning_rate.py

As an example you'll see the doc string from conditional gradient being generated for the web:
https://github.com/tensorflow/addons/blob/master/tensorflow_addons/optimizers/conditional_gradient.py
https://www.tensorflow.org/addons/api_docs/python/tfa/optimizers/ConditionalGradient

**Comment by sayakpaul** [comment]
Makes sense. Thanks, Sean.

**Comment by sayakpaul** [comment]
@seanpmorgan apologies for the reiteration. Would you rather prefer a small tutorial demoing CLR?

**Comment by seanpmorgan** [comment]
> @seanpmorgan apologies for the reiteration. Would you rather prefer a small tutorial demoing CLR?

Hmmm, If you think it's fundamental to usage it should go in docstring, but if it's more than that then the tutorials are also placed on the website:
https://www.tensorflow.org/addons/tutorials/optimizers_conditionalgradient

Perhaps both if it's not too much to ask? Thank you very much for your help.

**Comment by sayakpaul** [comment]
Sure thing. I will try to come up with a draft within a few days.
